client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -45,6 +53,27 @@ const PatternLibraryBody = styled.div``;
 
 export const patternFiltersClassName = 'pattern-library__filters';
 
+const ToggleGroupControlOptionWithNarrowTooltip = (
+	props: ToggleGroupControlOptionProps & {
+		className: string;
+		showToolTip?: boolean;
+		toolTipText: string;
+	}
+) => {
+	const { showToolTip, toolTipText } = props;
+	const toolTipProps = { style: { maxWidth: '200px' }, text: toolTipText };
+
+	if ( ! showToolTip ) {
+		return <ToggleGroupControlOption { ...props } />;
+	}
+
+	return (
+		<Tooltip { ...toolTipProps }>
+			<ToggleGroupControlOption { ...props } />
```

Comment:
I think we should limit which props are passed here. I'm seeing the following warning in the browser console:

```
React does not recognize the `toolTipText` prop on a DOM element
```

Reply:
Good catch